pyproject.toml

```diff
@@ -34,6 +34,7 @@ psutil = "^5.9.0"
 nest-asyncio = "^1.5.6"
 frozenlist = "^1.3.3"
 mkdocs-gallery = "0.7.6"
+xarray = ">=2022.3.0"
```

Comment:
Labeling array dimensions (i.e., with `xarray`) would be useful to simplify indexing and prevent `numpy` broadcasting bugs, which can be useful for a lot of multidimensional matrix calculations. Open to discussion about whether to introduce additional dependencies into this library though.